graphistry/Plottable.py

```diff
@@ -87,6 +87,18 @@ def get_topological_levels(
         raise RuntimeError('should not happen')
         return self
 
+    def collapse(
```

Comment:
Is there some sort of outer loop that is supposed to be used in a typical case? I'd think we'd want 2 methods:

1. clarify via renaming the current entrypoint as `collapse_adjacent_by_col_val_match(g, root, col, val)`
  - later, we cab generalize to the still-vectorizable form `collapse_adjacent_by_dict_match(g, root, {col1: val1, col2: val2, ...})`
2. add typical outer loop `collapse_adjacent_by_col_match(g, root, col)`
  - basically: `for v in g._nodes[col].unique(): g = collapse(g, root, col, v)`
  - later do the same generalization

We can land without for now..